lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile.py

```diff
@@ -171,26 +171,26 @@ def _get_network_id(self):
 
     def _get_qos_id(self):
         qoss_service = self._get_dcs_service().service(self._get_dcs_id()).qoss_service()
```

Comment:
Can you add `if self.param('qos')` at the top of the method, so we don't query `dcs`, in case it's not needed.